.github/CODEOWNERS

```diff
@@ -425,7 +425,7 @@
 /sdk/personalizer/                                                   @sharathmalladi
 
 # PRLabel: %Graph
-/sdk/graphrbac/                                                      @msyyc @ChenxiJiang333
+/sdk/graphrbac/                                                      @lmazuel
```

Comment:
This package is deprecated.

You can just delete the entry.

Reply:
Let's keep it so the bot can auto-label issues (line 427)